week-1-practise/level-1/10-function-output-3.js

```diff
@@ -1,5 +1,6 @@
 function concatenate(firstWord, secondWord, thirdWord) {
     // Write the body of this function to concatenate three words together
+    return firstWord.concat(secondWord).concat(thirdWord)
```

Comment:
Good stuff. You can also make things easier by passing multiple arguments to `concat`. Take a look a the docs here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#Syntax.

As an interesting note, there are multiple ways to do everything, and you also use the `+` to achieve the same result and there are slight differences in the way they work https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#Performance.

Reply:
Do you mean in this way:
function concatenate(firstWord, secondWord, thirdWord) {
return firstWord + secondWord + thirdWord;
}